internals/attachments.py

```diff
@@ -76,18 +62,17 @@ def store_attachment(feature_id: int, content: bytes, mime_type: str) -> str:
     # Create and save a thumbnail too.
     thumb_content = None
     try:
-      thumb_content = images.resize(content, THUMB_WIDTH, THUMB_HEIGHT)
-    except images.LargeImageError:
-      # Don't log the whole exception because we don't need to see
-      # this on the Cloud Error Reporting page.
-      logging.info('Got LargeImageError on image with %d bytes', len(content))
+      im = Image.open(io.BytesIO(content))
+      im.thumbnail((THUMB_WIDTH, THUMB_HEIGHT))
+      thumb_buffer = io.BytesIO()
+      im.save(thumb_buffer, 'PNG')
+      thumb_content = thumb_buffer.getvalue()
     except Exception as e:
       # Do not raise exception for incorrectly formed images.
-      # See https://bugs.chromium.org/p/monorail/issues/detail?id=597 for more
-      # detail.
       logging.exception(e)
     if thumb_content:
       attachment.thumbnail = thumb_content
+      logging.info('Thumbnail is %r bytes', len(thumb_content))
```

Comment:
Did you mean to leave this log statement here?

Reply:
Yes, I'd like to keep this for a while to make sure that things are working as intended.